src/java/com/media/FileProcessor.java

```diff
@@ -48,7 +57,8 @@ private FileProcessor(File rawDest, File standardDest, File mobileDest, List<Str
 		
 		this.commands = commands;
 	}
-	
+
+
 	/** Assumes that there will already be an image in /uploads/raw for it to find **/
 	public static FileProcessor ImageProcessor(File rawDest, File standardDest, File mobileDest) {
```

Comment:
Odd caps -- Why `ImageProcessor` not `imageProcessor`?

Reply:
No idea! I think Mark wrote this class. Amended.